### PR TITLE
Merging the 7.8.0 release into master

### PR DESCRIPTION
**Pull request by ryanwilson**

### CODE REVIEW

**Review by maksymmalyhin — APPROVED** [review]

File: Firestore/Swift/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# v7.7.0
```
Comment (paulb777) [inline Firestore/Swift/CHANGELOG.md:1]: should be 7.8.0

**Review by paulb777 — APPROVED** [review]
LGTM on nit fix and green

File: Firestore/Swift/CHANGELOG.md (hunk 1 above)
Comment (ryanwilson) [reply Firestore/Swift/CHANGELOG.md:1]: I'm fairly confident this was actually shipped in 7.7.0 but we missed the CHANGELOG update. 
Version 7.7.0 highlighted in the latest Firestore Swift PR

**Comment by paulb777** [comment]
Looks like the MLDownloader flake we've seen in the nightlies. I'm ok to consider it non-blocking here.

**Pull request merged**